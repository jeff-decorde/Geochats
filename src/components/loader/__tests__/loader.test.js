import Loader from '../loader.js';
import React from 'react';

describe('Loader', () => {
  const wrapper = shallow(<Loader />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot(wrapper);
  });
});
