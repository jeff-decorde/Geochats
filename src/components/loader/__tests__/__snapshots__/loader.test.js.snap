// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[object Object] 1`] = `
<div
  className="spinner"
>
  <i
    className="fas fa-spinner fa-pulse"
  />
</div>
`;
